src/runtime_src/core/pcie/windows/system_windows.cpp

```diff
@@ -130,6 +131,17 @@ get_os_info(boost::property_tree::ptree &pt)
   BufferSize = sizeof value;
   RegGetValueA(HKEY_LOCAL_MACHINE, "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion", "ProductName", RRF_RT_ANY, NULL, (PVOID)&value, &BufferSize);
   pt.put("distribution", value);
+
+  BufferSize = sizeof value;
+  RegGetValueA(HKEY_LOCAL_MACHINE, "SYSTEM\\CurrentControlSet\\Control\\SystemInformation", "SystemProductName", RRF_RT_ANY, NULL, (PVOID)&value, &BufferSize);
+  pt.put("model", value);
+
+  MEMORYSTATUSEX mem;
+  mem.dwLength = sizeof(mem);
+  GlobalMemoryStatusEx(&mem);
+  pt.put("memory", mem.ullTotalPhys / 0x100000);
```

Comment:
Same comment as comment regarding changing the JSON property name and how the value is formatted.

Reply:
Done